### ISSUE TITLE
Compiling under Linux?

### ISSUE THREAD

**Issue by amca01** [sinandredemption/zuuv#2]
According to the Readme file, Linux is no longer supported.  If that means that binaries are no longer distributed, that makes perfect sense.  But as gcc is fully supported under Linux, is it possible to compile any recent version of zuuv?  I would very much like to explore it, but I have no access to either Windows or MacOS (nor do I want to!).  Thank you very much.

**Comment by sinandredemption** [comment]
Zuuv is using y-cruncher's multi threaded multiplication libraries, which
unfortunately have a hard dependency on Windows. I loathe Windows so much
but unfortunately y-cruncher's multiplication routines are state-of-the-art
(and also the only multi threaded bignum C++ libraries I'm aware of
currently). Hence the Windows dependency.

You could port to Linux via substituting api calls to mpz equivalents or
so, but that would make a magnitude order of difference speed wise.
Nevertheless I did maintain something like that prior to this in the
previous version, so porting is possible.

**Comment by amca01** [comment]
Many thanks indeed.  However, I checked the y-cruncher page, and the newest version has been released in both Windows and Linux versions.  This should mean that it is possible (in Linux) to call the y-cruncher libraries as needed ... unless of course I'm missing something or misunderstanding something (both of which are quite likely)!

Thanks again.

**Comment by sinandredemption** [comment]
Right. However, the y-cruncher is a closed source program, and so is the
public interface to it's internals which Zuuv uses:
http://www.numberworld.org/ymp/v1.0/

Unfortunately, the author has abandoned this project since with no support
for Linux.

**Comment by amca01** [comment]
That's such a pity.  Oh well, I'll look elsewhere, or do what I can with the GMP/MPFR libraries and Julia.  Thanks very much for responding to my questions!
